### PR TITLE
add 3r new device（dual plug）

### PR DESCRIPTION
**Pull request by 3reality-support**

### CODE REVIEW

File: src/devices/third_reality.ts [hunk 1]
```diff
@@ -406,6 +406,74 @@ const definitions: DefinitionWithExtend[] = [
             }),
         ],
     },
+    {
+        zigbeeModel: ['3RDP01072Z'],
+        model: '3RDP01072Z',
+        vendor: 'Third Reality',
+        description: 'Zigbee / BLE dual plug with power',
```
Comment (Koenkk) [inline src/devices/third_reality.ts:413]: Can you remove everything below this line and try with:

```
ota: true
extend: [deviceEndpoints({endpoints: {left: 1, right: 2}}), onOff({endpointNames: ['left', 'right']}), electricityMeter({endpointNames: ['left', 'right']})],
```

Comment (3reality-support) [reply src/devices/third_reality.ts:423]: ok

Comment (Koenkk) [reply src/devices/third_reality.ts:423]: Can you now try after removing `fromZigbee`, `toZigbee` and `exposes`?

Comment (3reality-support) [reply src/devices/third_reality.ts:423]: ok

**Comment by Koenkk** [comment]
Thanks!

Could you also submit a picture for the docs? This can be done by:
1. Create a fork by clicking [here](https://github.com/Koenkk/zigbee2mqtt.io/fork)
2. Go to the `public/images/devices` directory, *Add file* -> *Upload files*
3. Upload the files and press *Commit changes*
4. Press *Contribute* -> *Open pull request* -> update title/description -> *Create pull request*

Make sure that:
- The filename is `3RDP01072Z.png`
- The size is 512x512
- The background is transparent (use e.g. [Adobe remove background](https://new.express.adobe.com/tools/remove-background))

**Pull request merged**